webrtchack/webrtchack.go

```diff
@@ -0,0 +1,83 @@
+// TODO: This needs to also be called for remotes
+package webrtchack
```

Comment:
I'd like to avoid introducing a top-level package. Can this live in gostream? Or camera? Or the slightly misnomered rdk/grpc package (webrtc.go)? without introducing a cycle.

Reply:
Yeah I intentionally named it something silly so we could have a convo on where it should live. 

I'll try to put it in: rdk/grpc  (as that is what it is most related to) and report back if it creates a cycle & we need to reconsider